src/ms.tex

```diff
@@ -925,14 +925,18 @@ \subsection{Project funding} \label{sec:project-funding}
 Project also received financial support from the Dunlap
 Seed Funding program at the University of Toronto to develop educational
 resources for scientific software packaging within the Learn Astropy
-framework. The project also indirectly benefited from the NRAO ALMA Development
+framework. Moreover, the Project indirectly benefited from the NRAO ALMA Development
```

Comment:
`Project` instead of `project` for consistency with the rest of the document. Added `Moreover` to avoid saying `The Project also` in two consecutive sentences.